src/utils/locale.js

```diff
@@ -25,8 +25,9 @@ export function resolveConfig(config, locales) {
   }
   id = id || detLocale;
   id = [id, id.substring(0, 2)].find(i => has(locales, i)) || detLocale;
-  // Spread the default locale to prevent repetitive update loops
-  const defLocale = { ...locales[id] };
+  // Add fallback and
+  // spread the default locale to prevent repetitive update loops
+  const defLocale = { ...locales['en-IE'], ...locales[id], id };
```

Comment:
i only used `en-IE` because the format is `DD-MM-YYYY` (which from what i have seen on the internet is more common than having the month come before the day) and has `dow: 2` which i assume is Monday.  
then again it is all up to you.